src/metrics/gameMetrics.ts

```diff
@@ -5,5 +5,6 @@ export const gamesPlayedMetric = new PromMetricCounter({
   help: 'Total number of games played.',
   labelOptions: {
     status: new Set(['finished', 'voided']),
+    game_type: new Set(['custom', 'matchmaking', 'private']),
```

Comment:
Let's call it something other than `game_type`. It can be easily confused with avalon / hunter or fab 4 or anything else. 

How's something like "room_creation_type"? 

On second thought, private games should probably be a separate metric. 

Match made games vs custom games.  
Within custom games, there's public and private. Idk on second thought if we should be mixing these into the same label.

Reply:
from previous discussions have done a room_type which is matchmaking, public and private.